workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSSendSystem.cs

```diff
@@ -7,45 +11,54 @@ namespace Improbable.Gdk.Core
     public class SpatialOSSendSystem : ComponentSystem
     {
         private WorkerBase worker;
-        private MutableView view;
 
-        private readonly List<int> registeredReplicators = new List<int>();
-        private readonly List<int> commandSenders = new List<int>();
+        private readonly List<SpecifiComponentReplicator> specificComponentReplicators =
+            new List<SpecifiComponentReplicator>();
 
         protected override void OnCreateManager(int capacity)
         {
             base.OnCreateManager(capacity);
 
             worker = WorkerRegistry.GetWorkerForWorld(World);
-            view = worker.View;
 
-            GenerateComponentGroups();
+            GetDefaultComponentReplicators();
         }
 
-        private void GenerateComponentGroups()
+        private void GetDefaultComponentReplicators()
```

Comment:
This does more then just `Getting`. Needs a better name.

Reply:
`PopulateDefaultComponentReplicators` ?

Reply:
Much better